src/main/java/seedu/lifeasier/notes/NoteCommandFunctions.java

```diff
@@ -0,0 +1,34 @@
+package seedu.lifeasier.notes;
+
+public class NoteCommandFunctions {
+
+    public static void checkEmptyList(NoteList notes) throws EmptyNoteListException {
+        if (notes.size() == 0) {
+            throw new EmptyNoteListException();
+        }
+    }
+
+    public static void printMultipleMatches(NoteList notes, String title) {
+        for (int i = 0; i < notes.size(); i++) {
+            if (notes.get(i).getTitle().contains(title)) {
+                System.out.println(i + 1 + ". " + notes.get(i).getTitle() + "\n");
```

Comment:
This line can be sent to Ui class - Pass in the "i" and the "note title" into a Ui method to print for you